gitlab-status/handler.go

```diff
@@ -6,11 +6,16 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"strings"
 
 	"github.com/alexellis/hmac"
 	"github.com/openfaas/openfaas-cloud/sdk"
 )
 
+const (
+	SystemSubDomain = "system"
```

Comment:
What is this? Perhaps it should not be hard-coded in a const but configured via environment like everything else?

Reply:
This is for the times the `gateway_public_url` is configured like `https://system.o6s.io` and we need `system` to replace the subdomain. I need to extend the functionality of this PR to support `pretty_url` environmental variable because I did not consider this. Totally forgot about this PR ..